ui/app/common/constants.js

```diff
@@ -9,6 +9,7 @@ Bahmni.Common = Bahmni.Common || {};
     var RESTWS = hostUrl + "/openmrs/ws/rest";
     var RESTWS_V1 = hostUrl + "/openmrs/ws/rest/v1";
     var BAHMNI_CORE = RESTWS_V1 + "/bahmnicore";
+    var BAHMNI_COMMONS = RESTWS_V1 + "/bahmni";
```

Comment:
@sivareddytw the key doesn't match with value. is the value should be "bahmnicommons" ?

Reply:
@binduak Angshu suggested to use the end point with `/bahmni` instead of `/bahmnicommons`.